### PR TITLE
{bio}[foss/2022a] kb-python v0.27.3 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by laraPPr**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by laraPPr** [comment]
bot: help

File: easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+sanity_pip_check = True
+
+exts_list = [
+    ('shortuuid', '1.0.11', {
+        'checksums': ['fc75f2615914815a8e4cb1501b3a513745cb66ef0fd5fc6fb9f8c3fa3481f789'],
+    }),
+    ('ngs-tools', '1.8.3', {
+        'checksums': ['c50e36105654cdea14b4c7971994a3e710157663aaf0fd72c4d83a486f12cdb0'],
+    }),
+    ('tenacity', '8.1.0', {
```
Comment (hajgato) [inline easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2022a.eb:34]: tenacity has a newer version. Is there any reason why it is not updated?

File: easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2022a.eb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        'checksums': ['e48c437fdf9340f5666b92cd7990e96bc5fc955e1298baf4a907e3972067a445'],
+    }),
+    ('plotly', '5.13.0', {
```
Comment (hajgato) [inline easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2022a.eb:37]: plotly has a newer version. Is there any reason why it is not updated?

File: easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2022a.eb [hunk 3]
```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'kb-python'
```
Comment (Micket) [inline easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2022a.eb:3]: We try to stick to the same naming as the project refers to itself. I would suggest `kallisto-bustools`

Comment (laraPPr) [reply easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2022a.eb:3]: Does this than have an impact on this easyconfig [kb-python-0.27.3-foss-2021b.eb](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2021b.eb)? Will the software than not be available under multiple names in easybuild?

**Review by hajgato — APPROVED** [review]

**Comment by hajgato** [comment]
@boegelbot: please test @ generoso

**Comment by hajgato** [comment]
@boegelbot please test @ generoso

**Comment by hajgato** [comment]
@Micket can you start tests for me? Thx in advance.

**Comment by robert-mijakovic** [comment]
@boegelbot please test @ generoso

**Comment by hajgato** [comment]
thx @robert-mijakovic

**Comment by hajgato** [comment]
Going in, thanks @laraPPr!

**Pull request merged**